### DISCUSSION TITLE
I don't understand the "objects" config...

### DISCUSSION THREAD

**Discussion by tve** [blakeblackshear/frigate#1371] [Ask A Question]
By default, `object` seems to have `track: [ 'person', 'car']`, but I want to track any animal. Can I set `track: []` for "anything" or does that track nothing? Do I have to go through the whole list of labels in the coco dataset and list all the ones I want in the `track` list?

Similar question for the default `filters: { person: { ...} }`, is that just a refinement for person, i.e., "if you track a person, apply this additional filter"? Or do I need a filter for every possible object? Is it OK to set `filters: {}` to remove all filtering?

Right now my camera is seeing motion but not recognizing anything. For example:
![image](https://user-images.githubusercontent.com/39480/125337074-67c06080-e303-11eb-9789-aecfa84fcc52.png)

**Comment by blakeblackshear** [comment]
You need to list the objects from the COCO dataset that you want to track. Birds are not in the default list. I don't recommend removing the default filters. You don't have to specify them for all objects.
